docs/_sass/_global.scss

```diff
@@ -81,6 +81,13 @@ img {
 	flex: 1;
 }
 
+@media (max-width: 767px) {
+  .container {
```

Comment:
In this file it's just the one line that has spaces instead of tabs - can you please update this as well?